k8s/README.md

```diff
@@ -12,12 +12,24 @@ kubectl apply -f contiv-vpp.yaml
 # undeploy
 kubectl delete -f contiv-vpp.yaml
 ```
-Optionaly you can edit `contiv-vpp.yaml` to deploy the dev-contiv-vswitch image, built
+Optionally you can edit `contiv-vpp.yaml` to deploy the dev-contiv-vswitch image, built
 in local environment with `../docker/build-all.sh`.
 ```
 sed -i "s@image: contivvpp/vswitch@image: dev-contiv-vswitch:<your image version>@g" ./contiv-vpp.yaml
 ```
 
+This manifest can be generated and updated from the contiv-vpp helm chart:
+```
+helm template contiv-vpp > contiv-vpp.yaml
```

Comment:
basic installations won't have tiller running until after the CNI plugin is working, we need to describe the template way with manifest and optionally leave the helm install (though I feel it's odd without instructions on how to get tiller running and the caveats)

Reply:
I added a little more text here and some installation notes in the contiv-vpp/README.md for installing via manifest or helm. Let me know if you had something else in mind.